docs/examples/numba_func/Numba_func.ipynb

```diff
@@ -0,0 +1,370 @@
+{
```

Comment:
<p>outn and inm reads a bit weird. Consider out_n, in_m.</p><p> </p><p>"By default, the numba function operator works on a per-sample basis. It's also possible to specify a function that processes a whole batch at once, by specifying the argument <code>batch_processing=True</code>. In this case, the outputs and inputs that the function receives contain a leading dimension, representing the sample index.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>